pkg/storage/node_liveness.go

```diff
@@ -692,13 +692,30 @@ func (nl *NodeLiveness) updateLivenessAttempt(
 	return nil
 }
 
+// maybeUpdate replaces the liveness (if it appears newer) and invokes the
+// registered callbacks if the node became live in the process.
 func (nl *NodeLiveness) maybeUpdate(new Liveness) {
 	nl.mu.Lock()
+	// Note that this works fine even if `old` is empty.
 	old := nl.mu.nodes[new.NodeID]
```

Comment:
Not a big deal by any means, but using `new` as the variable name confuses syntax highlighters :)

Reply:
Maybe your syntax highlighter... :-)

![image](https://user-images.githubusercontent.com/5076964/31735761-2cc0466e-b443-11e7-9630-c7ac1848b7b7.png)

Won't eat another CI, but I'll keep it in mind and rename the next time I touch this code (hopefully a while down the road).

Reply:
The github syntax highlighter is the one I was referring to :)

<img width="359" alt="screen shot 2017-10-18 at 2 35 45 pm" src="https://user-images.githubusercontent.com/7085343/31736077-b084c556-b411-11e7-8f8c-efe855138744.png">